src/transformState.js

```diff
@@ -62,7 +62,36 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    for (const key in transforms[i]) {
+      if (transforms[i][key] === 'addProperties') {
+        continue;
```

Comment:
that's fire, but there is a better approach, your solution use one loop and trying to handle different cases, which makes your code hard to read, would be better if you create different cases (switch-case construction), and run different loops(if it needed, depend on case)